mteb/leaderboard/benchmark_selector.py

```diff
@@ -64,6 +64,7 @@ class MenuEntry:
                         "MTEB(jpn, v1)",
                         "MTEB(kor, v1)",
                         "MTEB(pol, v1)",
+                        "MTEB(nld, v1)",
```

Comment:
```suggestion
```

Let us add it in a separate PR once we have results in as well (to avoid an empty benchmark